### ISSUE TITLE
Sync not enabled by default

### ISSUE THREAD

**Issue by mirageboss** [andryou/scriptsafe#216]
I think I speak for everyone when I say that Sync should be enabled by default when I open up Chrome on another computer that sync all my extensions etc.

**Comment by war59312** [comment]
It's disabled for privacy reason by default.

So it's not "calling home" without user permission and transferring private data without the user's consent (you have to first enable the setting).

**Comment by mirageboss** [comment]
Yeah, I get that, but it makes no sense to me.

If I am logged in to my Google Account on my browser, I want all my bookmarks, extensions and settings (even extension settings) to port over. I'm not talking about syncing the whole of the data in an insecure manner, I'm talking about enabling the sync option on all my devices if I have checked it and am syncing my extensions and extension data.

An example of this is how uBlock syncs the lists I have subscribed to, to every device...
